subprojects/delivery/cd/build.gradle.kts

```diff
@@ -6,31 +6,31 @@ plugins {
 
 dependencies {
     implementation(gradleApi())
-    implementation(projects.subprojects.testRunner.reportViewer)
-    implementation(projects.subprojects.gradle.android)
-    implementation(projects.subprojects.gradle.artifactoryAppBackup)
-    implementation(projects.subprojects.logger.gradleLogger)
     implementation(projects.subprojects.common.files)
     implementation(projects.subprojects.common.problem)
+    implementation(projects.subprojects.delivery.artifactoryAppBackup)
+    implementation(projects.subprojects.delivery.qapps)
+    implementation(projects.subprojects.delivery.signer)
+    implementation(projects.subprojects.delivery.uploadCdBuildResult)
+    implementation(projects.subprojects.delivery.uploadToGoogleplay)
+    implementation(projects.subprojects.gradle.android)
     implementation(projects.subprojects.gradle.buildEnvironment)
     implementation(projects.subprojects.gradle.git)
+    implementation(projects.subprojects.gradle.gradleExtensions)
     implementation(projects.subprojects.gradle.impact)
     implementation(projects.subprojects.gradle.impactShared)
-    implementation(projects.subprojects.testRunner.instrumentationTests)
-    implementation(projects.subprojects.gradle.gradleExtensions)
     implementation(projects.subprojects.gradle.prosector)
```

Comment:
What about this module?
It looks like a standalone check but in practice used only in the delivery process.